dockerfiles/base/Dockerfile

```diff
@@ -26,31 +26,49 @@ ARG USER_ID=802
 ARG USER_GROUP=wso2
```

Comment:
@RidmiR as per the requirements of the automated Docker image build process, IMO it would be better to avoid creating a base such as this and replicate the same instructions across all the profiles. We cannot introduce the logic of building a base (which is not to be pushed to the target registry and is a temporary image) and using that temporary base to build the other images into the automated build.

Please see the latest WSO2 Enterprise Integrator version `6.x.x` range Docker images for an example.

Reply:
@chirangaalwis I discussed about this with @DilanUA and did the changes.
@DilanUA need your thoughts on this